api/models/ml_model.py

```diff
@@ -59,23 +58,35 @@ def validate_image_size(cls, image_size):
     @root_validator(skip_on_failure=True)
     def check_models_and_device(cls, values):
         if values.get("device") == "Jetson":
-            if values.get("name") not in ["ssd_mobilenet_v2_coco", "ssd_mobilenet_v2_pedestrian_softbio",
-                                          "openpifpaf_tensorrt"]:
+            if values.get("name") not in MODELS_DEVICES["Jetson"]:
                 raise ValueError(f'The device {values.get("device")} only supports the following models:'
                                  f'"ssd_mobilenet_v2_coco", "ssd_mobilenet_v2_pedestrian_softbio",'
                                  f'"openpifpaf_tensorrt". ')
```

Comment:
Here you should print the `MODELS_DEVICES["Jetson"]` list, same in all cases

Reply:
Changed!